wzdx/combination/attenuator.py

```diff
@@ -0,0 +1,190 @@
+import json
+from datetime import datetime
+from wzdx.tools import cdot_geospatial_api, polygon_tools
+import logging
+
+ATTENUATOR_TIME_AHEAD_SECONDS = 30 * 60
+ISO_8601_FORMAT_STRING = "%Y-%m-%dT%H:%M:%SZ"
+
+
+def get_route_info_planned_event(planned_event):
+    start = cdot_geospatial_api.get_route_and_measure(
+        planned_event['geometry']['coordinates'][0])
+    end = cdot_geospatial_api.get_route_and_measure(
+        planned_event['geometry']['coordinates'][1])
+    if start['Route'] != end['Route'] or start['Measure'] == end['Measure']:
+        return None
+    else:
+        return {
+            'route': start['Route'],
+            'start_measure': start['Measure'],
+            'end_measure': end['Measure']
+        }
+
+
+def get_route_info_geotab(geotab):
```

Comment:
This method never seems to be called, do we need it?

Reply:
removed